app/routes/index.tsx

```diff
@@ -1,11 +1,52 @@
-import { LinksFunction } from '@remix-run/node';
+import { LinksFunction, LoaderFunction } from '@remix-run/node';
 import styles from 'react-datepicker/dist/react-datepicker.css';
 import Calendar from '~/components/Calendar';
 import Navbar from '~/components/common/navbar';
+import axios from 'axios';
+import moment from 'moment';
+import { useStore } from '~/store/useStore';
+import { useLoaderData } from '@remix-run/react';
+import { parseEvents } from '~/utils/event.utils';
+import { getEvents } from '~/constants/urls.constants';
+import { ToastContainer, toast } from 'react-toastify';
+import { useEffect } from 'react';
+
+export let loader: LoaderFunction = async ({ request }) => {
+  const cookie = request.headers.get('cookie');
+
+  const startTime = moment().subtract(1, 'months').startOf('month').unix() * 1000;
+  const endTime = moment().add(1, 'months').endOf('month').unix() * 1000;
+
+  try {
+    const response = await axios.get(getEvents(process.env.API_HOST ?? '', startTime, endTime), {
```

Comment:
we should create `axios wrappers` in `constants` file and just export the function

Reply:
@isVivek99 axios wrapper should be in a separate util file not the constants one. An issue is already raised for it in the repo.